tools/xcode-templates/install-xcode-templates.sh

```diff
@@ -1,16 +1,24 @@
 #!/bin/bash
 
 if [ ! -f "Package.swift" ]; then
-    echo "\`install-xcode-templates.sh\` must be run in repository root folder: \`./tools/xcode-templates/install-xcode-templates.sh\`"; exit 1
+    echo "\`install-xcode-templates.sh\` must be run in repository root folder: \`./tools/xcode-templates/install-xcode-templates.sh\`"
+    exit 1
 fi
 
 XCODE_LOCATION=$(xcode-select -p)
-XCODE_TEMPLATES_LOCATION="$XCODE_LOCATION/Library/Xcode/Templates/File Templates/"
+
+SOURCE_TEMPLATES_LOCATION="./tools/xcode-templates/Datadog/"
+TARGET_TEMPLATES_LOCATION="$HOME/Library/Developer/Xcode/Templates/File Templates/Datadog"
 
 if [ -z "$XCODE_LOCATION" ]; then
```

Comment:
Do we need this check now 🤔? Now it seems to be independent from the Xcode location, no?

Reply:
correct 👍 
removed ✅